src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1711,6 +1711,26 @@ export function registerTerminalActions() {
 			}
 		}
 	});
+	registerAction2(class extends Action2 {
+		constructor() {
+			super({
+				id: TerminalCommandId.ShowOrFocusHover,
+				title: terminalStrings.showOrFocusHover,
+				f1: false,
+				category,
+				precondition: ContextKeyExpr.or(ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated), TerminalContextKeys.isOpen),
+				keybinding: {
+					primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KeyK, KeyMod.CtrlCmd | KeyCode.KeyI),
+					weight: KeybindingWeight.WorkbenchContrib,
+					when: ContextKeyExpr.or(TerminalContextKeys.tabsFocus, TerminalContextKeys.focus)
+				}
+			});
+		}
+		async run(accessor: ServicesAccessor) {
+			const accessibilityService = accessor.get(IAccessibilityService);
+			accessor.get(ITerminalGroupService).showHover(accessibilityService.isScreenReaderOptimized());
```

Comment:
shouldn't this also happen for non-screen reader users? or at the least, be a part of the context key expression so ppl can use it if they want to?

Reply:
I was copying what show or focus hover seems to do now

Reply:
well, if this is strictly for screen reader users, the notion of `show` doesn't matter much anyway. But I think this could be useful for non-screen reader users who are used to the way this works in the editor

Reply:
Changing this to just focus as it improved the non-a11y mode experience too